nixos/modules/services/misc/workout-tracker.nix

```diff
@@ -0,0 +1,123 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.workout-tracker;
+in
+
+{
+  options = {
+    services.workout-tracker = {
+      enable = lib.mkEnableOption "A workout tracking web application for personal use (or family, friends), geared towards running and other GPX-based activities";
+
+      package = lib.mkPackageOption pkgs "workout-tracker" { };
+
+      user = lib.mkOption {
+        type = types.str;
+        default = "workout-tracker";
+        description = "User account under which Workout-tracker runs.";
+      };
+
+      group = lib.mkOption {
+        type = types.str;
+        default = "workout-tracker";
+        description = "Group under which Workout-tracker runs.";
+      };
+
+      environmentFile = lib.mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = "/run/keys/workout-tracker-env";
+        description = ''
+          An environment file as defined in {manpage}`systemd.exec(5)`.
+
+          Secrets like `WT_JWT_ENCRYPTION_KEY` may be passed to the service without adding them
+          to the world-readable Nix store.
+        '';
+      };
+
+      settings = lib.mkOption {
+        type = lib.types.submodule {
+          freeformType =
```

Comment:
Condensing this into one or two lines would make the code a lot more readable

Reply:
not sure what you mean by this. If you mean formatting, I'm using `nixfmt-rfc-style` which formats vertical space on its own.

Reply:
Also fixed.